src/themes/default/components/core/ProductListing.vue

```diff
@@ -11,7 +11,7 @@ import { coreComponent } from 'lib/themes'
 import ProductTile from './ProductTile.vue'
 
 export default {
-  props: ['products', 'columns'],
```

Comment:
We shouldn't remove this props, because we are using this component on other pages like home and product pages with different settings.

Reply:
@mercs600 the strange thing is that mixins (also i cant understand why not extends) merging props from "base component" which is core/ProductListing and i dont see any reason to doubleing them.
And the second one, check my little refactoring of this component(base)